src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -117,7 +117,10 @@ class PaymentMethodList extends Component {
      * @returns {Array}
      */
     getFilteredPaymentMethods() {
-        let combinedPaymentMethods = PaymentUtils.formatPaymentMethods(this.props.bankAccountList, this.props.cardList, this.props.payPalMeUsername, this.props.userWallet);
+        // filter non-billing cards
```

Comment:
Let's explain why we're filtering the billing card from the payment methods, something like:
```suggestion
        // Hide the billing card from the payments menu for now because you can't make it your default method, or delete it
```